src/main/java/org/eclipse/emfcloud/jackson/databind/property/EObjectFeatureProperty.java

```diff
@@ -43,21 +46,29 @@ public class EObjectFeatureProperty extends EObjectProperty {
    private final boolean defaultValues;
 
    private JsonSerializer<Object> serializer;
+   private JsonDeserializer<Object> deserializer;
```

Comment:
Please update the year in the copyright header to 2022, i.e., `2019-2022`.

Reply:
Done.

Reply:
Done.